README.md

```diff
@@ -1,25 +1,175 @@
+  For make this, first you should specifie car folder with its mac address :
+  > 📒 [ansible/config/hostnames](ansible/config/hostnames)
+  ```text
+  e4:5f:01:68:17:2c	exampleCar
+  ```
+  afterwards, just create a folder with the car name at `ansible/config`. For our example we should create exampleCar folder at `ansible/config/exampleCar`.
+
+  In this folder we should to create two file :
+   - `ds4drv.env` --> ps4 controller color
+   - `myconfig.py` --> donkeycar myconfig
+
+   example :
+   > 📒 [ansible/config/dababycar/ds4drv.env](ansible/config/dababycar/ds4drv.env)
+   ```env
+  CONTROLLER_LED_COLOR=A103FC
+   ```
+  
+  > 📒 [ansible/config/dababycar/myconfig.py](ansible/config/dababycar/myconfig.py)
+   ```python
+  DRIVE_TRAIN_TYPE = "PIGPIO_PWM" # SERVO_ESC|DC_STEER_THROTTLE|DC_TWO_WHEEL|SERVO_HBRIDGE_PWM|PIGPIO_PWM|MM1|MOCK
+
+  STEERING_CHANNEL = 12           #channel on the 9685 pwm board 0-15
+  STEERING_LEFT_PWM = 739         #pwm value for full left steering
+  STEERING_RIGHT_PWM = 400        #pwm value for full right steering
+  
+  STEERING_PWM_PIN = 12           #Pin numbering according to Broadcom numbers
+  STEERING_PWM_FREQ = 75          #Frequency for PWM
+  STEERING_PWM_INVERTED = False   #If PWM needs to be inverted
+
+  THROTTLE_CHANNEL = 13           #channel on the 9685 pwm board 0-15
+  THROTTLE_FORWARD_PWM = 575      #pwm value for max forward throttle max 750
+  THROTTLE_STOPPED_PWM = 470      #pwm value for no movement
+  THROTTLE_REVERSE_PWM = 400      #pwm value for max reverse throttle
+
+  THROTTLE_PWM_PIN = 13           #Pin numbering according to Broadcom numbers
+  THROTTLE_PWM_FREQ = 75          #Frequency for PWM
+  THROTTLE_PWM_INVERTED = False   #If PWM needs to be inverted
+
+  AUTO_RECORD_ON_THROTTLE = False #if true, we will record whenever throttle is not zero. if false, you must manually toggle recording with some other trigger. Usually circle button on joystick.
+  CONTROLLER_TYPE='custom' # Set the controller to be used to be our custom one ()
+
+  LOGGING_LEVEL='DEBUG'
+  ```
+
+## 🚀 Launch ansible recipe
+
+***
+
+After have create your ansible playbook, you can launch it with these two options :
+- install it by ssh but you should configure manually internet, ssh and install ansible on your raspberry pi. Follow this [tutorial](ansible/README.md)
+- generate your raspian image with donkeycar installed, follow this [tutorial](packer/README.md)
+
+  <br/>	
+</details>
+
+<details>	
+  <summary><b>✨ Our features</b></summary>
+  <br/>	
+ 
+## Software 
+
+***
+
+### Ansible remote installation documentation : 
+  > we have made a ansible book for automate our donleycar and raspberry pi installation remotely (ssh).
+    Indeed, these recipes allow us to have one single clean installation and be able to repeat it to infinity. <br>
+    <b>ℹ️ Read [documentation](ansible/README.md) !! </b>
+
+### Generate raspian image with donkeycar installation :
```

Comment:
Also IIRC in english you're not supposed to put a space before punctuation, but you can keep it if it looks better in the rendered view